data/en/resources.json

```diff
@@ -371,115 +373,117 @@
     "PackageManagementProvider": "NuGet",
     "AdditionalMetadata": {
       "summary": "The cNtfsAccessControl module contains DSC resources for NTFS access control management.",
+      "Authors": "Serge Nikalaichyk",
+      "DscResources": "cNtfsAuditEntry cNtfsAuditInheritance cNtfsPermissionEntry cNtfsPermissionsInheritance",
+      "lastUpdated": "30/04/2022 20:36:06 -07:00",
       "CLRVersion": "4.0",
+      "created": "06/02/2019 16:41:41 -08:00",
       "NormalizedVersion": "1.4.1",
+      "updated": "2022-04-30T20:36:06Z",
```

Comment:
Is it the format of the `updated` property it doesn't like? But the same format was used before, but other properties are using the format `30/04/2022 20:36:06 -07:00` 🤔